typescript.js

```diff
@@ -4,12 +4,16 @@ module.exports = {
    * */
   overrides: [
     {
-      files: ['*.ts', '*.tsx'],
+      files: ['*.ts?(x)'],
       parser: '@typescript-eslint/parser',
-      extends: ['plugin:@typescript-eslint/recommended', 'prettier/@typescript-eslint'],
+      parserOptions: { warnOnUnsupportedTypeScriptVersion: false },
```

Comment:
makes it so we don't see the silly warning when not on a specifically correct TS according to the parser.